lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/ts_lowering_context.cpp

```diff
@@ -0,0 +1,90 @@
+#include "lazy_tensor_core/csrc/ts_backend/ts_lowering_context.h"
+
+#include "lazy_tensor_core/csrc/compiler/node_lowering.h"
+
+namespace torch_lazy_tensors {
+namespace compiler {
+namespace ts_backend {
+
+TSLoweringContext::TSLoweringContext(const std::string& name, Device device)
+    : ir::LoweringContext(name, device),
+      graph_(std::make_shared<torch::jit::Graph>()) {}
+
+TSLoweringContext::TSLoweringContext(
+    const std::string& name, Device device,
+    absl::Span<const ir::Node* const> post_order,
+    ir::Util::EmissionMap emit_status)
+    : ir::LoweringContext(name, device, post_order, emit_status),
+      graph_(std::make_shared<torch::jit::Graph>()) {
+  auto lowering = NodeLowering::Create(this);
+  for (auto node : post_order) {
```

Comment:
why not use LowerNode(node) as below?

Reply:
It was a bad helper, I've removed it instead. It instantiated a new node lowering every time, which isn't a semantic problem since the lowering doesn't have state but still a janky way to do it.